src/setfit/modeling.py

```diff
@@ -179,12 +186,12 @@ def forward(
         if isinstance(features, dict):
             assert "sentence_embedding" in features
             is_features_dict = True
-
         x = features["sentence_embedding"] if is_features_dict else features
         logits = self.linear(x)
-        logits = logits / (temperature + self.eps)
-        probs = nn.functional.softmax(logits, dim=-1)
-
+        if self.multitarget or self.out_features == 1:  # only has one target or multiple targets per item
```

Comment:
`self.out_features == 1` is no longer used, see:
https://github.com/huggingface/setfit/blob/f777c2c60b270604dae0dc1db4eea815e8c9019d/src/setfit/modeling.py#L135-L139